internal/evaluator/conftest_evaluator.go

```diff
@@ -427,6 +427,19 @@ func addMetadataToResults(r *output.Result, rule rule.Info) {
 	if len(rule.Collections) > 0 {
 		r.Metadata[metadataCollections] = rule.Collections
 	}
+
+	if effectiveOnString, ok := r.Metadata["effective_on"]; ok {
+		effectiveOnTime, err := time.Parse(effectiveOnFormat, effectiveOnString.(string))
+		if err == nil {
+			// If the rule has been effective for a long time, we'll consider
+			// the effective_on date not relevant and not bother including it
+			if effectiveOnTime.Before(time.Now().Add(effectiveOnTimeout)) {
```

Comment:
I think `time.Now()` is wrong here FYI, you have to use `c.policy.EffectiveTime()` somehow.